medical/models/res_partner.py

```diff
@@ -48,26 +32,31 @@ class ResPartner(models.Model):
     alias = fields.Char(
         string='Alias',
         size=256,
-        help='Common name that the Party is reffered',
+        help='Common name the party is referred to as',
     )
     activation_date = fields.Date(
-        string='Activation date',
-        help='Date of activation of the party',
+        string='Activation Date',
+        help='Date the party was activated',
     )
     last_name = fields.Char(
         string='Last Name',
         size=256,
-        help='Last Name',
+        help='Last name of the party',
+    )
+    is_work = fields.Boolean(
+        string='Work',
+        help='Check if the party is a place of work',
     )
-    is_work = fields.Boolean(string='Work')
     is_person = fields.Boolean(
         string='Person',
-        help='Check if the party is a person.',
+        help='Check if the party is a person',
+    )
+    is_school = fields.Boolean(
```

Comment:
Add into roadmap (for v10):

Remove `is_*` attributes on in favor of `res.partner.type` model & a M2M relation